schema/field.go

```diff
@@ -124,6 +126,17 @@ func (schema *Schema) ParseField(fieldStruct reflect.StructField) *Field {
 		AutoIncrementIncrement: DefaultAutoIncrementIncrement,
 	}
 
+	if field.PrimaryKey {
+		field.PrimaryKeyPriority = 10
```

Comment:
Does this have any special meaning? Why is it 10?

Reply:
https://gorm.io/docs/indexes.html#Fields-Priority

> You can specify the order with the priority option, the default priority value is 10, if priority value is the same, the order will be based on model struct’s field index

The value 10 is intended to be the default value, similar to composite index.